.github/workflows/commit.yml

```diff
@@ -89,28 +90,34 @@ jobs:
       - name: Upload the SBOM
         uses: actions/upload-artifact@v3
         with:
-          name: SBOM - tool=${{ matrix.tool }}, environment=${{ matrix.environment }}
+          name: SBOM_${{ matrix.tool }}_${{ matrix.environment }}
```

Comment:
Renaming this is superficial but because it makes folders when downloading I wanted to get rid of characters that may need to be escaped/special cased.